content/community/steering/index.md

```diff
@@ -50,12 +50,13 @@ Please contact the committee member leading the project if you would like to lea
 
 * Membership - Dan Blankenberg
 * Misconceptions - Hans-Rudolf Hotz
-* Propeller onboarding cookbook - Anne Fouilloux
 * Global Science - Andrew Lonie
```

Comment:
@fubar2 do we move Global Science to the grown-up section too?

Reply:
It never really get going so perhaps not grown up? 
AFAIK Andrew Lonie won't mind.

Reply:
Links to each grown up project new home ?

Reply:
> Yes please. It never really get going so perhaps not grown up? AFAIK Andrew Lonie won't mind.

ok, then I'll remove it, thanks @fubar2

Reply:
> Links to each grown up project new home ?

they should be there already, is there anything missing?

Reply:
sorry, didn't look - distracted - now time to eat. Will look soonish.

Reply:
Looks good to me - thanks @beatrizserrano !

Reply:
Thank you to you both too!